### PR TITLE
Add io.sourceforge.clonekeenplus

### PR DESCRIPTION
**Pull request by gerstrong**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: io.sourceforge.clonekeenplus.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+command: CGeniusExe
+rename-desktop-file: cgenius.desktop
+rename-icon: cg
+
+modules:
+  - name: cgenius
+    buildsystem: cmake
+    sources:
+      - type: git
+        url: https://gitlab.com/Dringgstein/Commander-Genius.git
```
Comment (bilelmoussaoui) [inline io.sourceforge.clonekeenplus.yml:14]: This has to be a tagged release/tarball or a specific commit at least.

Comment (gerstrong) [reply io.sourceforge.clonekeenplus.yml:14]: You are right. Sorry for overseeing that. Just pushed another commit.

**Comment by bilelmoussaoui** [comment]
bot, build io.sourceforge.clonekeenplus

**Comment by bilelmoussaoui** [comment]
You need a appdata/metainfo file, see https://github.com/flathub/flathub/wiki/App-Requirements#appstream

**Comment by gerstrong** [comment]
> You need a appdata/metainfo file, see https://github.com/flathub/flathub/wiki/App-Requirements#appstream

Okay, got it. Thanks for your time. Now it should build correctly. Fingers crossed. :-)

**Comment by gerstrong** [comment]
bot, build io.sourceforge.clonekeenplus

**Comment by gerstrong** [comment]
Okay, just tested the build and it runs fine. Let me know if anything is missing for a successful publication. Thanks again for taking a look. :+1:

File: io.sourceforge.clonekeenplus.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      - install -D -m 644 -t $FLATPAK_DEST/share/icons/hicolor/scalable/apps ../share/cg.svg
+      - install -D -m 644 -t $FLATPAK_DEST/share/metainfo ../share/metainfo/io.sourceforge.clonekeenplus.appdata.xml
+
+finish-args: 
+  - --device=all # needed for gamepads 
+  - --share=ipc	 # needed for X11
+  - --share=network # needed for networking (Gamestore, downloads)
+  - --socket=x11
+  - --socket=pulseaudio
+  - --filesystem=home
```
Comment (barthalion) [inline io.sourceforge.clonekeenplus.yml:34]:
```suggestion
  - --filesystem=home
```

Does it make sense to expose entire home if it uses just `~/.CommanderGenius`?

File: io.sourceforge.clonekeenplus.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    post-install:
+      - for s in 16 32 48 64 128 256 512; do install -D -m 644 -t $FLATPAK_DEST/share/hicolor/${s}x${s}/apps ../share/cg.${s}.png; done
+      - install -D -m 644 -t $FLATPAK_DEST/share/icons/hicolor/scalable/apps ../share/cg.svg
+      - install -D -m 644 -t $FLATPAK_DEST/share/metainfo ../share/metainfo/io.sourceforge.clonekeenplus.appdata.xml
+
+finish-args: 
+  - --device=all # needed for gamepads 
+  - --share=ipc	 # needed for X11
+  - --share=network # needed for networking (Gamestore, downloads)
+  - --socket=x11
```
Comment (barthalion) [inline io.sourceforge.clonekeenplus.yml:32]: If it uses SDL2, I'd guess it works fine on Wayland too.

File: io.sourceforge.clonekeenplus.yml (hunk 2 above)
Comment (gerstrong) [reply io.sourceforge.clonekeenplus.yml]: Yes, just changed that and did a local test.

File: io.sourceforge.clonekeenplus.yml (hunk 3 above)
Comment (gerstrong) [reply io.sourceforge.clonekeenplus.yml:32]: also enabled wayland. Thank you.

**Comment by gerstrong** [comment]
bot, build io.sourceforge.clonekeenplus

**Pull request closed without merging**